flow/src/org/labkey/flow/controllers/FlowController.java

```diff
@@ -412,6 +391,23 @@ public void addNavTrail(NavTree root)
         }
     }
 
+    @RequiresPermission(AdminPermission.class)
+    public class MetricsAction extends ReadOnlyApiAction<Object>
```

Comment:
You may know this already, but you can get a preview of the full JSON metrics via Admin Console->Site Settings as well. No problems with having this action available too.